docs/site/Serving-static-files.md

```diff
@@ -0,0 +1,66 @@
+---
+lang: en
+title: 'Serving static files in LoopBack 4'
+keywords: LoopBack
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Serving-static-files.html
+---
+
+One of the basic requirements of a web app is the ability to serve static files.
+Serving static files from a LoopBack is very simple - just call the
+`app.static(path, rootDir[, options])` method. The variables in the API are
+explained below.
+
+- `app`: An instance of a LoopBack application.
+- `path`: The path where the static assets are to be served from. Refer to
```

Comment:
Should we perhaps use `urlPath` instead of `path`, to avoid possible confusion between `path` as a static asset configuration and `path` as the built-in Node.js module?

Reply:
Makes sense.